### PR TITLE
WTD AU 2018 recap

### PR DESCRIPTION
**Pull request by swapnilogale**
Added WTD 2018 recap

### CODE REVIEW

**Comment by swapnilogale** [comment]
@ericholscher @plaindocs - can you review/merge this? @bethaitman is planning to include this in the newsletter later today.

File: docs/conf/australia/2018/news/thanks-recap.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+- The main stage presentations had a good blend of topics across documentation, user experience, customer support, content strategy, coding and style guides.
+- The unconference sessions and the lightning talks was well received and prompted people to think, collaborate and talk about documentation in a more unstructured way. It also allowed people to discuss topics that were not covered fully or partially throughout the conference.
+- We received a mixed response to the workshops and this can be an area we can look at improving next year.
+Overall, we were so excited to see our first conference come together and see our awesome community gather under one roof.
+
+Conference venue
+================
+
+What better place to hold a documentation conference than a library! We had 2 rooms overlooking the sensational Port Phillip Bay right at the top floor of a public library. While the audience was not soaking in the documentation goodness, they were more than happy to stretch their legs and have a stroll through the library amidst tons of books.
```
Comment (plaindocs) [inline docs/conf/australia/2018/news/thanks-recap.rst]:
```suggestion
What better place to hold a documentation conference than a library! We had 2 rooms overlooking the sensational Port Phillip Bay right at the top floor of a public library. When the audience was not soaking in the documentation goodness, they were more than happy to stretch their legs and have a stroll through the library amidst tons of books.
```

File: docs/conf/australia/2018/news/thanks-recap.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Conscious consumption (aka Food and Drinks)
```
Comment (plaindocs) [inline docs/conf/australia/2018/news/thanks-recap.rst]: This made me double take, and I'm still not quite sure what you mean

File: docs/conf/australia/2018/news/thanks-recap.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+===========================================
+
+In the spirit of the community sharing nature of Write the Docs events, we decided to use social enterprises for our catering.
```
Comment (plaindocs) [inline docs/conf/australia/2018/news/thanks-recap.rst:33]: :+1:

File: docs/conf/australia/2018/news/thanks-recap.rst [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+When it came to beverages, we were ably supported by `Wheelly Good Coffee <https://wheellygoodcoffee.com.au/>`__, who have programs to improve the social and economic capabilities of some of Australia's most high risk young people.
+
+Videos and photos
+=================
+
+Videos of all the talks are now available on our `Youtube channel <https://www.youtube.com/playlist?list=PLy70RNJ7dYrJ1wANiqa7ObwUnoJjouQjt>`__.
+Thanks to `Darren Chan Photography <https://www.darrenchanphotography.com/>`__ (also one of our regular documentarians) for doing such a great job with these.
+You can also subscribe to our `YouTube channel <https://www.youtube.com/channel/UCPhWNEFb53x6PjnpRIYf1yg?>`_ so you don’t miss any future content!
+
+We also have photos from the event available on `Flickr <https://www.flickr.com/photos/writethedocs/albums/72157695294209550>`_, thanks to our wonderful photographer Darren Chan.
```
Comment (plaindocs) [inline docs/conf/australia/2018/news/thanks-recap.rst]: Double Darren Chan? Or is this a mixup?

**Review by plaindocs — COMMENTED** [review]
If you're feeling enthusiastic, the first couple of paragraphs could probably use some tweaking :-p

File: docs/conf/australia/2018/news/thanks-recap.rst (hunk 2 above)
Comment (swapnilogale) [reply docs/conf/australia/2018/news/thanks-recap.rst]: The library part? Let me revisit it.

File: docs/conf/australia/2018/news/thanks-recap.rst (hunk 4 above)
Comment (swapnilogale) [reply docs/conf/australia/2018/news/thanks-recap.rst]: No, the same person did both videos and photos

Comment (plaindocs) [reply docs/conf/australia/2018/news/thanks-recap.rst]: Might worth consolidating onto one sentence then? Two thanks feels odd to me

File: docs/conf/australia/2018/news/thanks-recap.rst (hunk 2 above)
Comment (plaindocs) [reply docs/conf/australia/2018/news/thanks-recap.rst]: > Conscious consumption

is what made me :confused:

Comment (swapnilogale) [reply docs/conf/australia/2018/news/thanks-recap.rst]: Maybe we could remove it entirely. It was only meant to say "whatever we consumed was with a social consciousness in mind"

Comment (plaindocs) [reply docs/conf/australia/2018/news/thanks-recap.rst]: :+1: following sentence addresses that nicely

File: docs/conf/australia/2018/news/thanks-recap.rst (hunk 4 above)
Comment (swapnilogale) [reply docs/conf/australia/2018/news/thanks-recap.rst]: Updated

**Comment by plaindocs** [comment]
Thanks @swapnilogale - wanna sort the conscious consumption heading, and I'll throw it in

**Review by plaindocs — APPROVED** [review]

**Comment by plaindocs** [comment]
@bethaitman  you're good to go!

**Comment by swapnilogale** [comment]
Cheers @plaindocs

**Comment by ericholscher** [comment]
Good stuff @swapnilogale & @plaindocs 💯

**Pull request merged**